docs/interfaces.md

```diff
@@ -460,6 +460,13 @@ processes on the system.
 
 * Auto-Connect: no
 
+### system-power-control
```

Comment:
I don't particularly care for `system-power-control` either since it is rather wordy. However, `shutdown-reboot` is weird. Perhaps `shutdown` or `shutdown-control` or even `power-off` is better. I don't think we necessarily need to split hairs on including reboot and/or shutdown in the interface name. Agreed @niemeyer should chat.